src/bio/smd/tests/smd_ut.c

```diff
@@ -224,12 +225,9 @@ ut_pool(void **state)
 		else
 			assert_true(false);
 
-		pool_cnt++;
-
 		d_list_del(&pool_info->spi_link);
 		smd_free_pool_info(pool_info);
 	}
-	assert_int_equal(pool_cnt, 2);
```

Comment:
It's better to keep this.

Reply:
agree with this.   Also it would be better to use a defined constant (or variable) rather than 2 (or whatever the new values should be